README.md

```diff
@@ -108,6 +108,11 @@ Currently, following dataset is supported.
 - QM9 [7, 8]
 - Tox21 [9]
 
+## Evaluation
+![tox21-evaluation](https://user-images.githubusercontent.com/29328746/39457687-974ad4ce-4d2a-11e8-9305-f6ca607af9dc.png)
+
+This image is the result of `examples/tox21/evaluate_models_tox21.sh`.
```

Comment:
I think we should note that we do not guarantee reproducibility of papers in which each model proposed and hence result doesn't imply performance of models themselves. For example, SchNet performs worst in the experiment, but it might be faults in our implementation.

Reply:
Yes I agree.

Reply:
@n-yoshikawa If you agree, could you add the aforementioned comments?

Reply:
I added the comment.